sass/styles/chart.scss

```diff
@@ -77,6 +77,12 @@ body {
     }
 }
 
+/*! rtl:ignore */
+.sharing .ciq-no-share {
```

Comment:
@ahmadtaimoor-deriv where are these classNames used? cannot find usages.
And if we are using them, where is it overriden ? why are we using `!important`'s ? :)

Reply:
Removed now , thanks